pkg/render/logstorage.go

```diff
@@ -354,37 +264,13 @@ func (es *elasticsearchComponent) Objects() ([]client.Object, []client.Object) {
 
 	toCreate = append(toCreate, es.elasticsearchCluster())
 
-	if es.cfg.KibanaEnabled {
-		// Kibana CRs
-		// In order to use restricted, we need to change elastic-internal-init-config:
-		// - securityContext.allowPrivilegeEscalation=false
-		// - securityContext.capabilities.drop=["ALL"]
-		// - securityContext.runAsNonRoot=true
-		// - securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost"
-		toCreate = append(toCreate, CreateNamespace(KibanaNamespace, es.cfg.Installation.KubernetesProvider, PSSBaseline))
-		toCreate = append(toCreate, es.kibanaAllowTigeraPolicy())
-		toCreate = append(toCreate, networkpolicy.AllowTigeraDefaultDeny(KibanaNamespace))
-		toCreate = append(toCreate, es.kibanaServiceAccount())
-
-		if len(es.cfg.PullSecrets) > 0 {
-			toCreate = append(toCreate, secret.ToRuntimeObjects(secret.CopyToNamespace(KibanaNamespace, es.cfg.PullSecrets...)...)...)
-		}
-
-		if len(es.kibanaSecrets) > 0 {
-			toCreate = append(toCreate, secret.ToRuntimeObjects(es.kibanaSecrets...)...)
+	if es.cfg.KeyStoreSecret != nil {
```

Comment:
This use to only be done when KibanaEnabled was false. Did you figure out it is ok to do this when KibanaEnabled is true?

Reply:
I think this was only enabled when FIPS was enabled. And for FIPS we had Kibana disabled. 

See the code in controller. 
https://github.com/tigera/operator/blob/master/pkg/controller/logstorage/elastic/elastic_controller.go#L405-L423